nca/NetworkConfig/NetworkConfigQuery.py

```diff
@@ -61,6 +64,49 @@ def policy_title(policy):
         """
         return f'{policy.policy_type_str()} {policy.full_name()}'
 
+    # following methods for writing query output in the required format
+    def write_query_output(self, query_answer, explanation_header='', yaml_explanation_descriptions=None, query_result=None):
+        """
+        calls the relevant method to write the query output in the required format
+        :param QueryAnswer query_answer: the query answer
+        :param str explanation_header: the txt message that describes the query's output explanation
+        :param list yaml_explanation_descriptions: [optional] the txt messages of explanations for yaml format,
+        relevant for TwoNetworkConfigsQuery classes
+        :param int query_result: [optional] the numerical result of the query, relevant for TwoNetworkConfigsQuery classes
+        """
+        # compatibility for the called methods
+        yaml_description = yaml_explanation_descriptions or explanation_header
+        numerical_result = query_result if query_result is not None else int(query_answer.numerical_result)
+        if self.output_config.outputFormat not in self.get_supported_output_formats():
+            return ''
+
+        if self.output_config.outputFormat == 'yaml':
+            return self.write_yaml_output(query_answer, yaml_description, numerical_result)
+
+        return self.write_txt_output(query_answer, explanation_header)
+
+    def write_txt_output(self, query_answer, explanation_description):
+        query_output = query_answer.output_result + '\n'
+        if query_answer.output_explanation:
+            query_output += self.compute_txt_explanation(query_answer, explanation_description) + '\n'
+        return query_output
+
+    def write_yaml_output(self, query_answer, descriptions, query_result):
+        query_name = self.output_config.queryName or type(self).__name__
+        configs_field = 'config' if isinstance(self, NetworkConfigQuery) else 'configs'
+        yaml_content = {'query': query_name, configs_field: self.get_configs_names()}
```

Comment:
I think that we can just use `configs `with a list of config names , whether it is 1 config or 2 configs.
`get_configs_names` would then return a list in both cases.
It will also simplify this code to one line:
`yaml_content = {'query': query_name, 'configs': self.get_configs_names()}`